testing/python/transform/test_tilelang_transform_pipeline_planning.py

```
@@ -754,6 +754,146 @@ def before(
+    assert len(stages) == 6
+    start_order, end_order, copy_order = orders[0], orders[1], orders[2]
+
+    assert stages[0] == stages[1] == stages[2] == 0
+    assert start_order < copy_order
+    assert end_order < copy_order
+    assert int(annos[0]["tl_pipelined_num_stages"]) == 1
+
+    # The first-stage group is emitted as one contiguous run after every
+    # consumer of the current iteration, so the single physical scalar is
+    # overwritten only once all of its readers have run.
+    first_stage_orders = sorted(o for s, o in zip(stages, orders) if s == 0)
```

Ruff (0.16.1)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)